nuxt.config.ts

```diff
@@ -228,6 +228,22 @@ const config: NuxtConfig = {
         ];
       }
     },
+    extend(config, { isDev, isClient }) {
```

Comment:
Linter's not happy

Reply:
I have no idea how to fix this, it's been like this since day one, as Nuxt generates this file this way.

Maybe @MrTimscampi has an idea.

Reply:
isDev can be removed
isClient is a bool

config is the webpack config object. Probably `any`. Not ideal though

Reply:
```suggestion
    extend(config: any, { isClient }: { isClient: boolean }): void {
```